apps/synapse-portal-framework/src/components/SectionLayout.tsx

```diff
@@ -24,6 +24,12 @@ export function SectionLayout(props: SectionLayoutProps) {
     ) : (
       <></>
     )
+  // Scroll to top if there's no hash in the URL
+  useEffect(() => {
+    if (!hash) {
+      window.scrollTo(0, 0)
+    }
+  }, [pathname, hash])
```

Comment:
A page may have many of these sections, so this may not the right place for this effect. But we can try it